### ISSUE TITLE
Add support for changing mode for all fans and pump mode in one line

### ISSUE THREAD

**Issue by cryptomilk** [audiohacked/OpenCorsairLink#202]
Hi,

I've looked into packaging OpenCorsairLink for Fedora. It would be possible to create a systemd service to execute it during boot to set the fan, led and pump modes. To achieve that I need to be able to execute OpenCorsairLink only once to set all the modes.

Example:

`OpenCorsairLink --device 0 --fan channel=0,mode=4 --fan channel=1,mode=4 --fan channel=2,mode=4 --pump mode=3 --led channel=0,mode=5`

Could you please add support for this?

Great that you use cmocka! ;-)

**Comment by cryptomilk** [comment]
Or a config file would be another option :-)

**Comment by darix** [comment]
Maybe with a simple format like this?

```yaml
devices:
- pump:
    mode: 3
  fan:
  - channel: 0
    mode: 4
  - channel: 1
    mode: 4
  - channel: 2
    mode: 44
  led:
  - channel: 0
    mode: 5
```

**Comment by audiohacked** [comment]
Systemd already supports a OneShot execution and also using multiple commands for starting a service.

**Comment by darix** [comment]
that is true but kinda ugly for a distro integration. do you really 
think offering the average user to edit service files is a good final 
solution?

**Comment by audiohacked** [comment]
Since most, if not all, init systems support executing multiple commands for starting a service. This request is out-of-scope for OpenCorsairLink.
